addons/slack/template/health.js

```diff
@@ -0,0 +1,21 @@
+// This function does a quick authenticated call to Slack to ensure the
+// token is valid. Customize this further to implement your own
+// health heuristic
+
+module.exports = async (slack, ctx) => {
+  try {
+    await slack.bot.api.test();
+  } catch (e) {
+    // Only pass through the error code and HTTP status
+    var result = {
+      body: { errorCode: e.code },
+      status: 500,
+    };
```

Comment:
status: e.statusCode

Reply:
The error won't always have a status code per their docs, hence I fall back to 500 here